packages/apollo-cache-control/src/index.ts

```diff
@@ -156,7 +156,7 @@ export class CacheControlExtension<TContext = any>
     let scope: CacheScope = CacheScope.Public;
 
     for (const hint of this.hints.values()) {
-      if (hint.maxAge) {
+      if (hint.maxAge !== undefined) {
         lowestMaxAge = lowestMaxAge
```

Comment:
Should the check here (before the `?`) also be against `undefined`?

Reply:
Yes, you are right. Did check that case. 😕 
Just fixed it and adjusted the test.